src/components/Tabs/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './Tabs';
+export * from './TabList';
+export * from './Tab';
+export * from './TabPanels';
+export * from './TabPanel';
```

Comment:
So you'll also need to expose this in the components index:

https://github.com/ModaOperandi/om/blob/master/src/components/index.ts

```ts
export { Tabs } from './Tabs';
```

Reply:
Done!